lib/LedgerSMB/Scripts/asset.pm

```diff
@@ -1010,7 +1010,18 @@ sub run_import {
 }
 
 ###TODO-LOCALIZE-DOLLAR-AT
```

Comment:
Actually, 2 lines below, you *have* localized "dollar-at" (`$@`), so this remark can go.